src/style.css

```diff
@@ -1,3 +1,74 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
Remember to use fallback fonts. In case the 'Roboto' font doesn't load, the text will be displayed in the default browser font, which may not be sans-serif. It's a good practice to specify a generic font family as a fallback. For example: `font-family: Roboto, Arial, sans-serif;`